swarm.go

```diff
@@ -122,7 +122,7 @@ func NewSwarm(config *api.Config, mockStore *mock.NodeStore) (self *Swarm, err e
 		HiveParams:   config.HiveParams,
 		LightNode:    config.LightNodeEnabled,
 		BootnodeMode: config.BootnodeMode,
-		SyncEnabled:  config.SyncEnabled,
+		SyncEnabled:  config.PullSyncEnabled, // should config.PushSyncEnabled have an effect here as well?
```

Comment:
here

Reply:
if you replicate the old behaviour you will have pull && push here

Reply:
thought this might be the case, thanks